docs/scripts/generate_plugin_docs.py

```
@@ -346,6 +351,150 @@ def fetch_github_stars(self, owner: str, repo: str) -> Optional[dict]:
+    ) -> Optional[List[dict]]:
+        """Fetch and parse manifest.json from a plugin repository."""
+        manifest_path = f"{path}/manifest.json" if path else "manifest.json"
+        url = f"https://raw.githubusercontent.com/{owner}/{repo}/{branch}/{manifest_path}"
+        try:
+            resp = requests.get(url, timeout=6)
+            if resp.status_code != 200:
+                return None
+            data = resp.json()
+            return data.get("models", [])
+        except (requests.RequestException, ValueError) as e:
+            logger.debug(f"No manifest found at {url}: {e}")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)